docs/deprecated.md

```diff
@@ -105,6 +106,26 @@ The table below provides an overview of the current status of deprecated feature
 | Removed    | [`--run` flag on `docker commit`](#--run-flag-on-docker-commit)                                                                    | v0.10      | v1.13   |
 | Removed    | [Three arguments form in `docker import`](#three-arguments-form-in-docker-import)                                                  | v0.6.7     | v1.12   |
 
+### OOM-score adjust for the daemon
+
+**Deprecated in Release: v24.0.0**
+**Target For Removal In Release: v25.0.0**
```

Comment:
```suggestion
**Subject for Removal in Release: v25.0.0**
```

Reply:
I'll keep this one to be the same, but we should indeed touch up this page a bit